subscription.js

```diff
@@ -102,17 +108,12 @@ class SubscriptionConnection {
       variables,
       operationName
     )
-      .then(async (result) => {
-        for await (const value of result) {
-          this.sendMessage(GQL_DATA, data.id, value)
-        }
-      })
-      .then(() => {
-        this.sendMessage(GQL_COMPLETE, data.id, null)
-      })
-      .catch(err => {
-        throw err
-      })
+
+    for await (const value of result) {
+      this.sendMessage(GQL_DATA, data.id, value)
```

Comment:
This does not implement any form of backpressure. Would you mind adding a TODO for this in the code? We might get back to it later in a follow-up PR.